### DISCUSSION TITLE
Can someone please review the documentation of the Tokio tutorial?

### DISCUSSION THREAD

**Discussion by CaliViking** [tokio-rs/tokio#5233] [General]
I posted on Discord about this as well.

The documentation for the Tokio tutorial is hard to read and understand. There is a combination of new concepts and writing code for the tutorial, it is not clear which is which. The code snippets are incomplete.

It may just be that I am new to the topic, but I would appreciate if someone other than the author could read the documentation and tell me if it makes sense to them as a tutorial.

Thanks in advance!

**Comment by matildasmeds** [comment]
Hi **CaliViking**! 
I know its been a while, but I think it is important feedback, and something I would like to try to help with. 
Is there some page that is particularly illustrative of the issues that you are highlighting? What would you expect to see in the tutorial?

**Comment by CaliViking** [comment]
I appreciate that you are reaching out. It has been so long that I no longer can remember the details.

Niels

**Reply by matildasmeds** [reply]
Got it, no worries about it. Should the distant memories gain form after all, please don't hesitate to add to the thread, open an issue or even submit a PR. That goes for everyone reading this too, please don't be shy.
Wishing you a great day!
